### PR TITLE
Test Entity > Chevron: Sent record with activated Toggle

### PR DESCRIPTION
**Pull request by EvgeniyaPiskunova**
US 
https://trello.com/c/HAi5Rkti/687-us165-entity-chevron-sent-record-with-activated-toggle

TC
https://trello.com/c/zDbaVZ1H/1134-tc165-entity-chevron-sent-record-with-activated-toggle

AT
https://trello.com/c/xumjTsUy/1145-at165-entity-chevron-sent-record-with-activated-toggle

### CODE REVIEW

File: src/test/java/EntityChevronSentRecordViaListTest.java [hunk 1]
```diff
@@ -76,4 +83,49 @@ public void testSentRecordViaList() {
...
+
+    private void clickSaveButton() {
+        findElement(SAVE_BUTTON).click();
+        getWait().until(ExpectedConditions.elementToBeClickable(findElement(CREATE_NEW_FOLDER)));
+    }
+
+    private void clickToggle() {
+        findElement(TOGGLE).click();
+    }
+}
```
Comment (MichaelMicrof) [inline src/test/java/EntityChevronSentRecordViaListTest.java:131]: add empty line here

File: src/test/java/EntityChevronSentRecordViaListTest.java [hunk 2]
```diff
@@ -76,4 +83,49 @@ public void testSentRecordViaList() {
...
+        clickSaveButton();
+        clickToggle();
+        jsClick(getDriver(), getDriver().findElement(
+                By.xpath("//button[text()='Sent']")));
+        jsClick(getDriver(), getDriver().findElement(
+                By.xpath("//a[contains(@href,'index.php?action=action_list&list_type=table&entity_id=36&stage=Sent')]")));
+
+
+        getWait().until(ExpectedConditions.textToBePresentInElementLocated(
+                By.xpath("//span[@class='pagination-info']"),
```
Comment (MichaelMicrof) [inline src/test/java/EntityChevronSentRecordViaListTest.java:101]: all selectors must be move to the beginning of the class

File: src/test/java/EntityChevronSentRecordViaListTest.java (hunk 1 above)
Comment (SergeiDemyanenko) [reply src/test/java/EntityChevronSentRecordViaListTest.java:131]: what for?

File: src/test/java/EntityChevronSentRecordViaListTest.java (hunk 2 above)
Comment (SergeiDemyanenko) [reply src/test/java/EntityChevronSentRecordViaListTest.java:101]: what for?

**Review by SergeiDemyanenko — APPROVED** [review]

**Pull request merged**